app/controllers/katello/api/v2/organizations_controller.rb

```diff
@@ -134,7 +134,7 @@ def releases
 
     api :PUT, "/organizations/:id/cdn_configuration", N_("Update the CDN configuration")
     param :id, String, :desc => N_("ID of the Organization"), :required => true
-    param :type, String, :desc => N_("CDN configuration type. One of %s.") % CdnConfiguration::TYPES, :required => true
+    param :type, String, :desc => N_("CDN configuration type. One of %s.") % CdnConfiguration::TYPES.join(", "), :required => true
```

Comment:
This change isn't compatible with 6.10, so it'll raise a merge conflict downstream. Not a big deal i guess..